services/src/pro/datasets/external/mod.rs

```diff
@@ -1 +1,70 @@
-pub mod sentinel_s2_l2a_cogs;
+mod sentinel_s2_l2a_cogs;
+
+use crate::api::model::datatypes::DataProviderId;
+use crate::error::Result;
+use crate::layers::external::{DataProvider, DataProviderDefinition};
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+
+pub use sentinel_s2_l2a_cogs::{
+    Band, GdalRetries, SentinelDataset, SentinelS2L2ACogsProviderDefinition,
+    SentinelS2L2aCogsDataProvider, SentinelS2L2aCogsMetaData, StacApiRetries, Zone,
+};
+
+#[derive(PartialEq, Debug, Serialize, Deserialize, Clone)]
+#[serde(tag = "type")] // TODO: rename_all = "camelCase"
```

Comment:
why the todo?

Reply:
We have everything in camelCase but these providers.